src/types/error.type.ts

```diff
@@ -0,0 +1,29 @@
+/* --- Error Types Definitions
+
+export class <ErrorName>Error extends Error {
```

Comment:
comment left intentionally?

Reply:
In the end this could be an npm script, like `npm run new:error --ProfileNotFoundError` to create a new error type from a template, but we'd need an autoloader for that (refer to command loading), keeping all errors separately.